validseqparenthesis.py

```diff
@@ -0,0 +1,26 @@
+def isvalid1(string):
+    dict1={'(':')','[':']','{':'}'}
+    stack=[]
+
+    for i in range(0,len(string)):
+        print(stack)
+        j=len(stack)
```

Comment:
instead of checking the length every time, initialize the variable j = 0 and increment it with every append and decrement with every pop.

This 1 statement increased your complexity from n to n square